source/utils/Helper.cpp

```diff
@@ -274,11 +274,16 @@ namespace Helper
     {
         return TTostring(d);
     }
-    #ifndef __x86_64
+    #if !defined(__LP32__) || defined(__OpenBSD__)
```

Comment:
`__LP32__` doesn't exist and `!defined(__ILP32__)` would break x86_64 again. `size_t` is `uint64_t` everywhere but on OpenBSD.

Reply:
The only alternative I see is then creating special rules for OpenBSD and general rules for the rest. __LP32__ does not really exist just realise clang create this macro possibly.

Reply:
If build broke because `size_t` != `uint64_t` on 64bit platform then `toString(size_t d)` is useful.